features.go

```diff
@@ -64,6 +64,11 @@ var featuresCommand = cli.Command{
 					},
 				},
 			},
+			PotentiallyUnsafeConfigAnnotations: []string{
+				"bundle",
```

Comment:
I totally forgot about this internal annotation. We should probably see if we can remove it...

(I suspect we also always override it, so it's technically not unsafe but it's better to be safe than sorry.)